src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -750,6 +753,118 @@ describe('<ResourceList />', () => {
       resourceList.update();
       expect(resourceList.find(BulkActions).prop('selectMode')).toBe(false);
     });
+
+    describe('focus', () => {
+      describe('large screen', () => {
+        it('focuses the checkbox in the bulk action when the plain CheckableButton is clicked', () => {
+          const resourceList = mountWithAppProvider(
+            <ResourceList
+              items={itemsWithID}
+              renderItem={renderItem}
+              promotedBulkActions={promotedBulkActions}
+            />,
+          );
+
+          const selectAllCheckableButton = resourceList.findWhere(
+            (wrap) => wrap.is(CheckableButton) && wrap.prop('plain') === true,
+          );
+
+          trigger(selectAllCheckableButton, 'onToggleAll');
+
+          const deselectAllCheckbox = resourceList
+            .findWhere(
+              (wrap) => wrap.is(CheckableButton) && !wrap.prop('plain'),
+            )
+            .find('input[type="checkbox"]');
+
+          expect(deselectAllCheckbox.getDOMNode()).toBe(document.activeElement);
+        });
+
+        it('focuses the plain CheckableButton checkbox when items are selected and the deselect Checkable button the is clicked', () => {
+          const resourceList = mountWithAppProvider(
+            <ResourceList
+              items={itemsWithID}
+              renderItem={renderItem}
+              selectedItems={allSelectedIDs}
+              promotedBulkActions={promotedBulkActions}
+            />,
+          );
+
+          const deselectAllCheckableButton = resourceList.findWhere(
+            (wrap) => wrap.is(CheckableButton) && !wrap.prop('plain'),
+          );
+
+          trigger(deselectAllCheckableButton, 'onToggleAll');
+
+          const selectAllCheckableCheckbox = resourceList
+            .findWhere(
+              (wrap) => wrap.is(CheckableButton) && wrap.prop('plain') === true,
```

Comment:
Same comment as a few . above

```suggestion
              (wrap) => wrap.is(CheckableButton) && wrap.prop('plain'),
```